### ISSUE TITLE
Consider removing `graphql-config-extension-graphcool` and `graphql-config-extension-prisma` from `graphql-playground-html`. 

### ISSUE THREAD

**Issue by wtgtybhertgeghgtwtg** [graphcool/graphql-playground#493]
#### This issue pertains to the following package(s):

- [ ] GraphQL Playground - Electron App
- [x] GraphQL Playground HTML
- [ ] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

#### What OS and OS version are you experiencing the issue(s) on?
N/A

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?
1.4.4

#### What is the expected behavior?
`graphql-playground-html` should not package 3rd party configuration parsers.

#### What is the actual behavior?
`graphql-playground-html` packages 3rd party configuration parsers.

#### What steps may we take to reproduce the behavior?
Look at the [`package.json`](https://github.com/graphcool/graphql-playground/blob/6c038af9332966ca29500d5d49ac74f4bb2b108b/packages/graphql-playground-html/package.json#L39-L40).

_Please provide a gif or image of the issue for a quicker response/fix._

In short:
 - Users of `graphql-playground` are not necessarily users of `graphcool` or `prisma`.
 - There is no documentation for using `graphcool.yml` or `prisma.yml` for configuration.
 - Configuration would be expected to come from arguments or `graphql-config`.
 - `graphql-config-extension-graphcool` and `graphql-config-extension-prisma` increase the dependency tree by a non-trivial amount.  Actually, most of the dependency tree is from those two and their transitive dependencies.

**Comment by schickling** [comment]
Thanks for bringing this up @wtgtybhertgeghgtwtg. While we won't remove the extensions directly, we'll definitely try to minimize to the dependency footprint.

We're also evaluating a plugin based architecture for v2. This might be a good opportunity to tackle this again.

**Comment by wtgtybhertgeghgtwtg** [comment]
I don't mind the footprint so much as the fact that it is unexpected, undocumented, and a little confusing.  What can be set from `.graphqlconfig` as opposed to `graphcool.yml`, `prisma.yml`, or `.env`?  What takes precedence?

**Comment by schickling** [comment]
Great point. We're actually currently working on the [documentation for GraphQL Playground](https://www.prismagraphql.com/docs/graphql-ecosystem/graphql-playground/overview-chaha125ho) where we should make sure to include this. cc @DevanB

**Comment by timsuchanek** [comment]
@wtgtybhertgeghgtwtg we removed these dependencies from the `graphql-playground-html` package completely. For users who need these, they can use the graphql cli.

**Comment by wtgtybhertgeghgtwtg** [comment]
Thank you.
